### ISSUE TITLE
Radarr installation fails on DS214play & DSM 7.0.X

### ISSUE THREAD

**Issue by Rel0ne** [SynoCommunity/spksrc#5363]
### Is this a new Bug?

- [X] I checkd that the bug hasn't been reported before

### Package Name

Radarr

### Package Version

v 3.0.0.4204

### Device Model

ds214play

### Device Architecture

i686 (evansport)

### Firmware Version

7.0.1-42218 Update 5

### What happened?

when i try to install the package i get the following error: "Invalid file format. Please contact package developer"

### Reproduction steps

```bash
1. search the package in package center
2. press install button
3. receive the error "Invalid file format. Please contact package developer"
```

### Install Log

```log
radarr.log does not exist in /var/log/packages/
```

**Comment by Joryzoetendaal** [comment]
I have exactly the same issue

**Comment by publicarray** [comment]
Unfortunately evansport is a 32 bit CPU and is no longer supported. Not even by the new beta packages: https://wiki.servarr.com/synology-packages
https://github.com/Servarr/spksrc/releases

I think the old Radarr version might be from when it was a mono package, and it wasn't made compatible with DSM7.0 since Radarr dropped mono support around the same time. It might be possible to create an old mono version for yourself and make it DSM 7.0 compatible, but I think you may have to do it yourself.

**Comment by publicarray** [comment]
Actually there was some work being done to support x84 https://github.com/SynoCommunity/spksrc/issues/5130#issuecomment-1102724813

**Comment by Rel0ne** [comment]
none yet, i offered my support for testing in the tread pointed by @publicarray ; strange is that Sonarr and Jackett works on DS241play but radarr won't start, even if I force installed x86 version

**Comment by paxx-cyber** [comment]
> none yet, i offered my support for testing in the tread pointed by @publicarray ; strange is that Sonarr and Jackett works on DS241play but radarr won't start, even if I force installed x86 version

Please let me know, if you have more information.

**Comment by bakerboy448** [comment]
x86 radarr support was released in spring and with 4.2.4 master

the packages will need updating to add support for x86
https://github.com/Radarr/Radarr/releases/tag/v4.2.4.6635

**Comment by mreid-tt** [comment]
I've done a test build in and the links to test are in this https://github.com/SynoCommunity/spksrc/issues/5130#issuecomment-1378736213.
